packages/backend/webpack.config.js

```diff
@@ -46,6 +46,16 @@ const config = {
           flags: "g",
         },
       },
+      {
+        test: /yeoman-environment[/|\\]lib[/|\\]namespace.js/,
+        loader: "string-replace-loader",
+        options: {
+          search: "namespace && namespace[.]constructor && namespace[.]constructor[.]name === 'YeomanNamespace'",
+          replace:
+            "namespace && namespace.constructor && (namespace.constructor.name === 'YeomanNamespace' || namespace.constructor.name === 'e' || (namespace._original && namespace.unscoped && namespace.generator !== undefined))",
```

Comment:
`namespace.constructor.name === 'e'`

Make me suspect you want to disable naming mangling in the bundling.
Note that the `.name === 'e'` is only true for this specific bundling with this specific
set of dependencies, it might be `d` or `f` after we upgrade a random dep...

Because this is a *.vsix you might even want to consider disabling minimifcation all-together....
- Verify the impact on *.vsix size...

- Possible config to start investigating
- https://webpack.js.org/plugins/terser-webpack-plugin/#terseroptions

<img width="802" height="920" alt="image" src="https://github.com/user-attachments/assets/d0674879-ffce-4bf2-8c93-e1d6b8338a06" />